src/update/UpdateManager.cpp

```diff
@@ -51,10 +56,26 @@ void UpdateManager::Start()
     auto& instance = GetInstance();
 
     static std::once_flag flag;
+
     std::call_once(flag, [&instance] {
         instance.mTimer.SetOwner(&instance, ID_TIMER);
         instance.mTimer.StartOnce(1);
         });
+
+    // Show the dialog only once. 
+    if (!prefUpdatesNoticeShown.ReadWithDefault(false))
```

Comment:
MIght just be `Read` because false IS the default specified for that settings object